internal/provider/resource_gitlab_label.go

```diff
@@ -20,6 +22,12 @@ var _ = registerResource("gitlab_label", func() *schema.Resource {
 		ReadContext:   resourceGitlabLabelRead,
 		UpdateContext: resourceGitlabLabelUpdate,
 		DeleteContext: resourceGitlabLabelDelete,
+		// FIXME: this importer sucks a little, but we can't use a passthrough importer, because
+		//        the resource id is flawed and we don't want to break backwards-compatibility.
```

Comment:
Looks like my earlier comment is something you're already aware of 😄

Reply:
Yeah, exactly - it's actually not the only resource -.^ ... It's something for v4 of this provider to change ..